### PR TITLE
file added boostrapassign1.html

### PR DESCRIPTION
**Pull request by vijaya1811**

### CODE REVIEW

File: PAWAN/BOOSTRAP/assignment1/boostrapassign1.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            top: 5px;
+            left: 20px;
+            color: rgb(235, 59, 132);
+        }
+        .border_text {
+            border: 1px solid black;
+            width: 100px;
+            height: 150px;
+            margin-left: 100px;
+        }
```
Comment (shubhamdalal49) [inline PAWAN/BOOSTRAP/assignment1/boostrapassign1.html]: use style tag inside the head tag

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

File: PAWAN/BOOSTRAP/assignment1/boostrapassign1.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" />
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+     </head>
+    <style background-image: url (image/pexels.jpg width= "800" height= "500");>
```
Comment (nileshp1122) [inline PAWAN/BOOSTRAP/assignment1/boostrapassign1.html]: style tag use in head

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

File: PAWAN/BOOSTRAP/assignment1/boostrapassign1.html (hunk 1 above)
Comment (vijaya1811) [reply PAWAN/BOOSTRAP/assignment1/boostrapassign1.html]: I done changes please kindly check

File: PAWAN/BOOSTRAP/assignment1/boostrapassign1.html [hunk 3]
```diff
@@ -7,8 +7,9 @@
         <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
         <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
         <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+        <link rel="stylesheet" href="assign1.css" />
      </head>
-    <style background-image: url (image/pexels.jpg width= "800" height= "500");>
+    <style>
```
Comment (nileshp1122) [inline PAWAN/BOOSTRAP/assignment1/boostrapassign1.html:11]: style tag place in head tag

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

File: PAWAN/BOOSTRAP/assignment1/boostrapassign1.html (hunk 2 above)
Comment (vijaya1811) [reply PAWAN/BOOSTRAP/assignment1/boostrapassign1.html]: yes I done changes please review

File: PAWAN/BOOSTRAP/assignment1/boostrapassign1.html (hunk 3 above)
Comment (vijaya1811) [reply PAWAN/BOOSTRAP/assignment1/boostrapassign1.html:11]: yes done

**Review by nileshp1122 — APPROVED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

**Review by urmilagagare — APPROVED** [review]

File: PAWAN/BOOSTRAP/assignment1/boostrapassign1.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+                        <div class="card-body">
+                            <img src="image/circle.png" style="width: 50px; height: 50px" />
+                            Pawan Jagtap posted a new assignment: Evalution 26 may 2021.
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </container>
+    </body>
+</html>
```
Comment (Pawan1005) [inline PAWAN/BOOSTRAP/assignment1/boostrapassign1.html:71]: make indentation foe head tag

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**